app/views/internal/broadcasts/index.html.erb

```diff
@@ -31,6 +31,7 @@
           <p>
             <%= broadcast.processed_html %>
           </p>
+          <h3>Last Active On: <%= broadcast.active_status_updated_at&.strftime("%b %d, %Y %H:%M UTC") %></h3>
```

Comment:
Might want to reword this description for clarity

Reply:
My concern with rewording this is that it is too lengthy. 😬  "Active Status Updated On:" is slightly clearer but is pretty long with the timestamp, especially compared to the rest of the text in the `/internal/broadcasts` view and "Active Status Updated:" doesn't read as well as "Last Active On:" or "Active Status Updated On:". That being said, I'm happy to change it if you have a better suggestion @mstruve! 😄

Reply:
gotcha, this works, if people get confused in the future we can always change it. As this starts to be used by others I think we will be documenting how all this works and we can use docs to clarify if we need to.

Reply:
Sounds like a plan! 👍 👍